extensions/helix/package.build.ts

```diff
@@ -137,6 +143,100 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+                ];
+
+                return [key, { command: "dance.run", args: { commands }, text: description }];
+              }),
+          );
+        })(),
+      },
+
+      resurround: {
+        title: "Resurround object...",
+        items: (() => {
+          const pairs = {
+            "()": "parenthesis block",
+            "{}": "braces block",
+            "[]": "brackets block",
+            "<>": "angle block",
+            '"': "double quote string",
+            "'": "single quote string",
+            "`": "grave quote string",
+          };
+          const enclosing = ["\\(", "\\)", "\\{", "\\}", "\\[", "\\]", "<", ">", '"', '"', "'", "'", "`", "`"];
+
+          return Object.fromEntries(
+            Object.entries(pairs)
+              .map(([key, description]) => {
+                const [open, close] = key.length === 1 ? [key, key] : Array.from(key);
+                const commands = [
+                  { command: "dance.seek.enclosing", args: { pairs: enclosing } },
+                  { command: "dance.edit.insert", args: { text: open, where: "start" } },
+                  { command: "dance.edit.insert", args: { text: close, where: "end" } },
+                  { command: "dance.selections.reduce.edges" },
+                  { command: "dance.edit.delete" },
+                  { command: "dance.selections.clear.secondary" },
+                ];
+
+                return [key, { command: "dance.run", args: { commands }, text: description }];
+              }),
+          );
+        })(),
+      },
+
+      unsurround: {
+        title: "Unsurround object...",
+        items: (() => {
+          const pairs = {
+            "m": { description: "enclosing pair", input: undefined },
+            "()": { description: "parenthesis block", input: "\\((?#inner)\\)" },
+            "{}": { description: "braces block", input: "\\{(?#inner)\\}" },
+            "[]": { description: "brackets block", input: "\\[(?#inner)\\]" },
+            "<>": { description: "angle block", input: "<(?#inner)>" },
+            '"': { description: "double quote string", input: '(?#noescape)"(?#inner)(?#noescape)"' },
+            "'": { description: "single quote string", input: "(?#noescape)'(?#inner)(?#noescape)'" },
+            "`": { description: "grave quote string", input: "(?#noescape)`(?#inner)(?#noescape)`" },
+          };
+
+          return Object.fromEntries(
+            Object.entries(pairs)
+              .map(([key, { description, input }]) => {
+                const seekCommand = input === undefined
+                  ? { command: "dance.seek.enclosing" }
+                  : { command: "dance.seek.object", args: [{ input }] };
+                const commands = [
+                  seekCommand,
+                    { command: "dance.selections.reduce.edges" },
+                    { command: "dance.edit.delete" },
+                    { command: "dance.selections.clear.secondary" },
```

Comment:
Sorry for the nit, last thing:
```suggestion
                  { command: "dance.selections.reduce.edges" },
                  { command: "dance.edit.delete" },
                  { command: "dance.selections.clear.secondary" },
```